lcserve/flow.py

```diff
@@ -351,6 +351,8 @@ def get_global_jcloud_args(app_id: str = None, name: str = APP_NAME) -> Dict:
                 },
                 'metrics': {
                     'enable': True,
+                    'host': 'http://opentelemetry-collector.monitor.svc.cluster.local',
```

Comment:
Why is this needed on the client side?

Reply:
because by default `metrics` isn't enabled and would require host/port according to @tarrantro

Reply:
That's because we user prometheus for metrics, otlp metrics is not stable as @winstonww mentioned. So we disable it to avoid duplicate metrics.